assets/src/edit-story/components/form/numeric.js

```diff
@@ -83,6 +83,35 @@ function Numeric({
   const [dot, setDot] = useState(false);
   const ref = useRef();
 
+  const altPressed =
+    float && useIsKeyPressed(ref, { key: 'alt', editable: true });
```

Comment:
Don’t call Hooks inside loops, conditions, or nested functions. https://reactjs.org/docs/hooks-rules.html.

Unfortunately the linter isn't currently catching this, but working on it: #1465 

I'd do `const altPressed = useIsKeyPressed(ref, { key: 'alt', editable: true });` and then do the `float` check could be inside the callback.

Reply:
That makes me think: should we just update `createKeyHandler` to properly pass the keyboard shortcut that was used to the callback. See "Checking keyboard combination from callback" section in the [docs](https://craig.is/killing/mice).

Then there would be no need for doing `useIsKeyPressed`.

Reply:
@dvoytenko thoughts on that?

Reply:
Changed to just use `createKeyHandler`. Did not find any documentation on how to access the `alt` property in the callback. But turned out to be simple enough :)